orangecontrib/text/nyt.py

```diff
@@ -51,7 +51,7 @@ def keywords(doc, name):
         (data.StringVariable('Creative Works'), lambda doc: NYT.keywords(doc, 'creative_works')),
         (tv, lambda doc: NYT.tv.parse(doc.get('pub_date'))),
         (data.DiscreteVariable('Article Type'), lambda doc: doc.get('type_of_material', None)),
-        (data.DiscreteVariable('Word Count'), lambda doc: doc.get('word_count', None)),
+        (data.StringVariable('Word Count'), lambda doc: doc.get('word_count', None)),
```

Comment:
Firstly I did change it to Continuous. But then I did not like the decimals shown. The easiest way to remove those zeros was changing Continuous to StringVariable.